src/routes/auth.js

```diff
@@ -102,4 +109,44 @@ router.post(
 	})
 )
 
+router.post(
+	'/token',
+	isAuthenticated(),
+	validateSchema(authTokenSchema),
+
+	catchAsync(async (req, res) => {
+		const { email ,  
+			refreshToken : currentRefreshToken 
+		 } = req.body
+
+		const userDetails = await userSevice.getNewTokens(
+			email,
+			currentRefreshToken
+		)
+
+		res.status(200).json({
+			status: 'success',
+			data: {
```

Comment:
use the same transformer function we created

Reply:
also kindly double check what the function you're calling in the service returns